app/frontend/src/features/communities/events/EventForm.tsx

```diff
@@ -0,0 +1,257 @@
+        className={classes.imageInput}
+        control={control}
+        id="event-image-input"
+        name="eventImage"
+        type="rect"
+      />
+      <PageTitle>{title}</PageTitle>
+      {error || errors.eventImage || errors.location ? (
+        <Alert severity="error">
+          {error?.message ||
+            errors.eventImage?.message ||
+            // @ts-expect-error
+            errors.location?.message ||
+            ""}
+        </Alert>
+      ) : (
+        isMutationSuccess && <Alert severity="success">{successMessage}</Alert>
+      )}
+      <form className={classes.form} onSubmit={onSubmit}>
+        <TextField
+          fullWidth
+          id="title"
+          inputRef={register({ required: TITLE_REQUIRED })}
+          name="title"
+          label={TITLE}
+          variant="standard"
+        />
+        <div className={classes.duoContainer}>
+          <Datepicker
+            control={control}
+            // @ts-expect-error
+            error={!!errors.startDate?.message}
+            // @ts-expect-error
+            helperText={errors.startDate?.message || ""}
+            id="startDate"
+            label={START_DATE}
+            name="startDate"
+          />
+          <Timepicker
+            control={control}
+            id="startTime"
+            label={START_TIME}
+            name="startTime"
+            // @ts-expect-error
+            error={!!errors.startTime?.message}
+            // @ts-expect-error
+            errorText={errors.startTime?.message || ""}
+          />
+        </div>
+        <div className={classes.duoContainer}>
+          <Datepicker
+            control={control}
+            // @ts-expect-error
+            error={!!errors.endDate?.message}
+            // @ts-expect-error
+            helperText={errors.endDate?.message || ""}
+            id="endDate"
+            label={END_DATE}
+            name="endDate"
+          />
+          <Timepicker
+            control={control}
+            id="endTime"
+            label={END_TIME}
+            name="endTime"
+            // @ts-expect-error
+            error={!!errors.endTime?.message}
+            // @ts-expect-error
+            errorText={errors.endTime?.message || ""}
+          />
+        </div>
+        <div
+          className={classNames(
+            classes.duoContainer,
+            classes.locationContainer
+          )}
+        >
+          {isOnline ? (
+            <TextField
+              error={!!errors.link?.message}
+              helperText={errors.link?.message || ""}
+              fullWidth
+              id="link"
+              name="link"
+              inputRef={register({ required: LINK_REQUIRED })}
+              label={VIRTUAL_EVENT_LINK}
+              variant="standard"
+            />
+          ) : (
+            <LocationAutocomplete
+              control={control}
+              fullWidth
+              label={LOCATION}
+              required={LOCATION_REQUIRED}
```

Comment:
This is currently stripping parts of the address for the profile editing. Probably needs a prop to disable this.

Reply:
Hmm interesting, considering I barely changed the implementation of this apart from adding an optional required prop, so it should be the same as before if nothing gets passed in that prop like in all other places.

What locations were you typing in? I can't seem to reproduce it on the preview build

Reply:
It is the same as before, but before it strips the street number and name from the address (right?). I tried something like 432 12th Street New York

Reply:
Oh I misread this and see what you mean now. The search results that come up here does only seem to show the stripped versions of the address yeah so yeah needs some tweaking...